source/androidx.core/core/Additions/AndroidX.Core.View.NestedScrollingParentHelper.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Collections.Generic;
+using Android.Runtime;
+
+namespace AndroidX.Core.View
+{
+	public partial class NestedScrollingParentHelper
+	{
+		public virtual void OnNestedScrollAccepted (Android.Views.View child, Android.Views.View target, int axis, int @type)
+		{
+			Android.Views.ScrollAxis scroll_axis = (Android.Views.ScrollAxis) axis;
+
+			OnNestedScrollAccepted(child, target, scroll_axis, @type);
+
+			return;
+		}
+
+		public virtual void OnNestedScrollAccepted(Android.Views.View child, Android.Views.View target, int axes)
+		{
+			Android.Views.ScrollAxis scroll_axis = (Android.Views.ScrollAxis) axes;
+
+			OnNestedScrollAccepted(child, target, scroll_axis);
+
+			return;
+		}
+
+		//public bool OnStartNestedScroll (Android.Views.View child, Android.Views.View target, int axes, int @type)
```

Comment:
Is this commented code intentional?

Reply:
Kinda.

I copied the code from one artifact to another and there were still `Removed methods` - because of parameter names.

Cleaning up.